lib/vrl/web-playground/public/vrl-highlighter.js

```diff
@@ -0,0 +1,636 @@
+            [
+                /}/,
+                {
+                    cases: {
+                    "$S2==interpolatedstring": {
+                        token: "string",
+                        next: "@pop",
+                    },
+                    "@default": "@brackets",
+                    },
+                },
+            ],
+            [/[{}()\[\]]/, "@brackets"],
+            [/@symbols/, "delimiter"],
+
+            // numbers
+            [/\d+[eE]([\-+]?\d+)?/, "number.float"],
+            [/\d+\.\d+([eE][\-+]?\d+)?/, "number.float"],
+            [/0[xX][0-9a-fA-F]+/, "number.hex"],
+            [/0[0-7]+(?!\d)/, "number.octal"],
+            [/\d+/, "number"],
+
+            // delimiter: after number because of .\d floats
+            [/[,.]/, "delimiter"],
+
+            // strings:
+            [/"""/, "string", '@herestring."""'],
+            [/'''/, "string", "@herestring.'''"],
+            [
+                /"/,
+                {
+                    cases: {
+                    "@eos": "string",
+                    "@default": { token: "string", next: '@string."' },
+                    },
+                },
+            ],
+            [
+                /'/,
+                {
+                    cases: {
+                    "@eos": "string",
+                    "@default": { token: "string", next: "@string.'" },
+                    },
+                },
+            ],
+        ],
+
+        string: [
+            [/[^"'\#\\]+/, "string"],
+            [/@escapes/, "string.escape"],
+            [/\./, "string.escape.invalid"],
+            [/\./, "string.escape.invalid"],
+
+            [
+                /#{/,
+                {
+                    cases: {
+                    '$S2=="': {
+                        token: "string",
+                        next: "root.interpolatedstring",
+                    },
+                    "@default": "string",
+                    },
+                },
+            ],
+
+            [
+                /["']/,
+                {
+                    cases: {
+                    "$#==$S2": { token: "string", next: "@pop" },
+                    "@default": "string",
+                    },
+                },
+            ],
+            [/#/, "string"],
+        ],
+
+        herestring: [
+            [
+                /("""|''')/,
+                {
+                    cases: {
+                    "$1==$S2": { token: "string", next: "@pop" },
+                    "@default": "string",
+                    },
+                },
+            ],
+            [/[^#\\'"]+/, "string"],
+            [/['"]+/, "string"],
+            [/@escapes/, "string.escape"],
+            [/\./, "string.escape.invalid"],
+
+            [/#{/, { token: "string.quote", next: "root.interpolatedstring" }],
+            [/#/, "string"],
+        ],
+
+        comment: [
+            [/[^#]+/, "comment"],
+            [/###/, "comment", "@pop"],
```

Comment:
We don't support this style (currently, it would highlight `###` as a comment, and whatever follows it as regular syntax, which is inacurate).